slick/src/main/scala/slick/jdbc/HsqldbProfile.scala

```diff
@@ -65,6 +66,13 @@ trait HsqldbProfile extends JdbcProfile {
 
   override val scalarFrom = Some("(VALUES (0))")
 
+  def defaultSqlTypeName(tmd: JdbcType[_], sym: Option[FieldSymbol]): String = tmd.sqlType match {
```

Comment:
Not compiling
```override def defaultSqlTypeName``` ?

It would be nice to also have a test for this in [GeneratedCodeTest.scala#L68](https://github.com/slick/slick/blob/4b6f6f2637128b38734f60dab07efe57d5027eed/slick-testkit/src/test/scala/slick/test/codegen/GeneratedCodeTest.scala#L68) to verify if the correct sqltype is set from generated code.
You'd also need to add an entry to [/dbs/hsqldb.sql](https://github.com/slick/slick/blob/master/slick-testkit/src/codegen/resources/dbs/hsqldb.sql)